### ISSUE TITLE
Force Sit stops working after relog

### ISSUE THREAD

**Issue by Shylei** [OpenCollarTeam/OpenCollar#533]
What version of OpenCollar are you using? 8.0.5

What behavior did you expect? To be returned to Force Sitting after relog

What behavior did you see instead? When Force Sit is applied, after relog it doesn't re-sit me on the object.

What steps does someone need to take to reproduce the problem?  Please try Force sit and relog to see the problem - other collars like Marine's or AC Juran's work fine.

**Comment by SilkieSabra** [comment]
Fprce sit and Forbid Stand are two different commands.  If Strict Sit is enabled, Forbid Stand is enforced when Force Sit is called.  Did you have.  In OpenCollar this does not persist after a relog and to my knowledge never has.  If you think OpenCollar should have this feature, rephrase this as a feature request.
